### ISSUE TITLE
Correct error: Cognito provider with no groups

### ISSUE THREAD

**Issue by impalah** [impalah/auth-middleware#14]
When a Cognito user has no groups assigned and an id_token is provided it throws the error:

```
DEBUG     2024-04-07 16:01:07,474 auth_middleware Get Current Active User ...
ERROR     2024-04-07 16:01:07,475 auth_middleware Not controlled exception 'scope'
ERROR     2024-04-07 16:01:07,475 auth_middleware Error in AuthMiddleware: 'scope'
```

Correct message strings.

**Comment by impalah** [comment]
Corrected on issue #23

**Comment by impalah** [comment]
Corrected on issue #23
